test/fw/ptl/utils/plugins/ptl_test_data.py

```diff
@@ -151,9 +151,19 @@ def __save_home(self, test, status, err=None):
             return
 
         servers = getattr(_test, 'servers', None)
+        if servers is not None:
+            server_host = servers.values()[0].shortname
+        else:
+            _msg = 'Could not find Server Object in given test object'
+            _msg += ', skipping saving post analysis data'
+            f.write(_msg + '\n')
+            self.logger.warning(_msg)
+            f.close()
+            return
         moms = getattr(_test, 'moms', None)
         comms = getattr(_test, 'comms', None)
         client = getattr(_test.servers, 'client', None)
+        server = servers.values()[0]
         add_hosts = []
         if len(servers) > 1:
             for param in servers.values():
```

Comment:
add_hosts will still add servers.values()[0], please replace 'servers' here with 'servers[1:]'

Reply:
I am going to address this comment. and also I will be removing the line add_hosts = list(set(add_hosts) - set(current_host)) as now pbs_snapshot is run irrespective of current_host.

Reply:
ya but if somebody passes 'servers=<remote_host1>,<remote_host2>', then servers[0] will not be "current_host" right? In fact I think that can be quite useful, being able to capture data from remote hosts while running pbs_snapshot on the local client host.

Reply:
I have changed current_host to server_host to avoid duplicates.